dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/JacksonUtils.java

```diff
@@ -86,4 +89,16 @@
             throw new UncheckedIOException( ex );
         }
     }
+
+    public static Map<String, Object> fromJsonToMap( String object )
+        throws IOException
```

Comment:
This becomes a bit too specific / not generic. Use generics, e.g. Map<T, U> mapFromJson. Then we can reuse later.